models/cohere.py

```diff
@@ -17,11 +17,21 @@ def coherence(con_caption: str, unc_caption: str, ocr_output: list[str]) -> str:
                 f"adhere to the following guidelines to ensure the quality and relevance of your output: Generalize "
```

Comment:
An example of cleaning up the grammar would be here on lines 17 and 18. You use colons frequently where they aren't needed. End the following sentence with a period, and _then_ start listing out the tasks, "Please adhere to the following guidelines to ensure the quality and relevance of your output**_._**"